main/src/ca/uwaterloo/flix/util/Formatter.scala

```diff
@@ -32,7 +32,7 @@ trait Formatter {
     def leftline: String = {
       val sb = new StringBuilder()
       for (lineNo <- beginLine to endLine) {
-        val currentLine = lineAt(lineNo)
```

Comment:
Why is this change also here? :)

Reply:
When writing `Result.try` I encountered a bug that crashed the compiler which was traced to that line. I decided to also put in the other PR as it is more relevant there. Maybe it should just be in its own PR. I think `lineAt` is type String but I can't recall atm. Will have a look at it tomorrow